civihr_default_theme/template.php

```diff
@@ -413,6 +406,23 @@ function _get_cog_menu_items() {
 }
 
 /**
+ * Builds and returns the markup of the given menu item
+ *
+ * @param array $menuItem
+ *
```

Comment:
There should not be a line break here.

Reply:
In PHP you need to leave new lines between `@params` and `@return`